src/index.ts

```diff
@@ -370,6 +374,24 @@ export default class Browserbase {
     await browser.close()
     return screenshot
   }
+
+  async createContext(): Promise<{ id: string }> {
+    const response = await fetch(`${this.baseAPIURL}/v1/contexts`, {
+      method: 'POST',
+      headers: {
+        'x-bb-api-key': this.apiKey,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ projectId: this.projectId }),
+    });
+  
+    if (!response.ok) {
+      throw new Error('Failed to create context');
+    }
+  
+    const data = await response.json();
+    return { id: data.id };
```

Comment:
```suggestion
    return data;
```

return the rest of the [response](https://docs.browserbase.com/reference/api/create-a-context#:~:text=in%20Settings.-,Response,-201%20%2D%20application/json) in case someone wants to upload a custom context